options/locale/locale_en-US.ini

```diff
@@ -1882,6 +1882,7 @@ wiki.page_title = Page title
 wiki.page_content = Page content
 wiki.default_commit_message = Write a note about this page update (optional).
 wiki.save_page = Save Page
+wiki.cancel = Cancel
```

Comment:
Occurrence number 12 of the regex `= Cancel$` in this file :tada:
Jokes aside, should we perhaps use one common key for it instead of adding the same key every time?

Reply:
Definitely! That's the plan for the locale cleanup anyway